src/angular.prefix

```diff
@@ -3,4 +3,4 @@
  * (c) 2010-2016 Google, Inc. http://angularjs.org
  * License: MIT
  */
-(function(window, document, undefined) {
+(function(window) {
```

Comment:
What would it guard against, though? We've used to have the same parameter in jQuery but we removed it; if someone created a variable `undefined` that is not equal to `undefined`, basically everything would be broken.